Makefile

```diff
@@ -1,8 +1,8 @@
-all     :; SOLC_FLAGS="--optimize --optimize-runs=1" \
+all     :; DAPP_BUILD_OPTIMIZE="1" SOLC_FLAGS="--optimize --optimize-runs=1" \
                dapp --use solc:0.6.11 build
 clean   :; dapp clean
 test    :; ./test-dssspell.sh
-deploy  :; SOLC_FLAGS="--optimize --optimize-runs=1" \
+deploy  :; DAPP_BUILD_OPTIMIZE="1" SOLC_FLAGS="--optimize --optimize-runs=1" \
     dapp --use solc:0.6.11 build && \
-    dapp create DssSpell --gas=${ETH_GAS} --gas-price=${ETH_GAS_PRICE}
+    dapp create DssSpell
```

Comment:
Hmm why aren't they necessary? and what do you mean with not working?

Reply:
I suspect these may have been deprecated in the new version Sam was using as well.

Reply:
Maybe it's another issue with the new version of dapptools, but I'm getting an error when those are present. The environment variables get picked up by the tool regardless, so I thought it would be okay to omit it.

Reply:
Ok will investigate it further next week